src/Libraries/RevitNodes/Elements/StucturalFraming.cs

```diff
@@ -247,17 +247,28 @@ private void InternalSetCurve(Autodesk.Revit.DB.Curve crv)
         /// <summary>
         /// Gets curve geometry from location of the specified structural element
         /// </summary>
-        new public Autodesk.DesignScript.Geometry.Curve Location
+        public new Autodesk.DesignScript.Geometry.Geometry Location
```

Comment:
I am wondering why the base class AbstractFamilyInstance don't have an abstract property to return Geometry rather than Point? Not sure how new property behaves in VM, FFI.